src/types/occupation.ts

```diff
@@ -0,0 +1,3 @@
+import { MyInfoField, StringValue, CodeAndDesc } from "./base"
+
+export type MyInfoOccupation = MyInfoField<StringValue | CodeAndDesc>
```

Comment:
@mantariksh do u know why this is possibly a `StringValue`? is it in case of custom occupation provided by a citizen?

Reply:
that would be my guess - FIN is govt-verified which would be code/desc, and SC/PR are user-provided which would be StringValue

<img width="700" alt="Screenshot 2021-03-04 at 2 36 55 PM" src="https://user-images.githubusercontent.com/29480346/109921599-1372f780-7cf7-11eb-8f71-d429190a2a00.png">

Reply:
OK, I extended the type system with 8bc85c46f4e24ae93fd31f150a1805ae1b4cdee8 to take care of this.

Reply:
We still don't know enough to push down the type for `MyInfoUnavailable` yet, but this should be pretty good.

Reply:
I don't suppose you know of any other fields that has categorical data if govt verified, but free-form if user-provided?

Reply:
nope, I think Occupation is the only one